src/routes/clubs/[clubId]/challenge/[challengeId]/members/+page.svelte

```
@@ -1,7 +1,13 @@
 <script lang="ts">
-	let { data } = $props();
+	import { page } from '$app/state';
+	import { getChallengeMembersData } from '$lib/remote/challenge.remote';
 
-	let { members, challengePath, admins } = $derived(data);
+	const data = await getChallengeMembersData({
+		clubId: page.params.clubId ?? '',
+		challengeId: page.params.challengeId ?? ''
+	});
```

ESLint

[error] Cannot use `await` in deriveds and template expressions, or at the top level of a component, unless the `experimental.async` compiler option is `true`
https://svelte.dev/e/experimental_async(experimental_async)

(svelte/valid-compile)